Rock.Messaging.RabbitMQ/RabbitEngine.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Threading.Tasks;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rock.Messaging.RabbitMQ
+{
+    public class RabbitEngine : IReceiver, ISender
+    {
+        private readonly IRabbitSessionConfiguration config;
+        private ConnectionFactory connectionFactory;
+        private IConnection connection;
+        private List<IModel> consumerModels; // While this is hacky- IModel is not thread safe.. see: https://www.rabbitmq.com/releases/rabbitmq-dotnet-client/v1.7.1/rabbitmq-dotnet-client-1.7.1-user-guide.pdf
```

Comment:
What exactly is this for? I see it initialized to an empty list, and then used in the Dispose, but nothing is ever added to it.